charts/rucio-server/templates/service.yaml

```diff
@@ -15,10 +15,7 @@ metadata:
 spec:
   type: {{ .Values.service.type }}
   ports:
-    - port: {{ .Values.service.port }}
```

Comment:
Can this just be replaced? Is this backwards compatible like that to not using nodeports?